latin_squares.py

```diff
@@ -133,226 +129,196 @@ def resample(self):  # TODO
         print("Done!")
         return
 
-    def get_lambda(self):
+    def ess_form(self, delta):  # DONE
+        V = np.array([self.V(p) for p in self.particles])
+        w = np.exp(- delta * V)
+        return np.sum(w)**2 / np.sum(w**2)
```

Comment:
Might be more efficient to let
`w = np.exp( - 2 * delta * V)`

Reply:
yeah, feel free to change

Reply:
I think it's easier for you to do it before I pull your changes in. Sorry ;)

Reply:
Yeah, sure